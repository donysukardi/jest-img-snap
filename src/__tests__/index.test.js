const puppeteer = require('puppeteer');

describe('index', () => {
  let browser;

  beforeAll(async () => {
    browser = await puppeteer.launch();
  });

  it('works', async () => {
    const page = await browser.newPage();
    await page.goto('https://www.google.com.sg/search?q=time&oq=time&aqs=chrome..69i57j69i61j69i59j69i61j0l2.2487j0j7&sourceid=chrome&ie=UTF-8');
    const img = await page.screenshot();
    expect(img).toMatchImageSnapshot();
    expect(img).toMatchImageSnapshot();
  }, 10000);

  afterAll(async () => {
    await browser.close();
  });
});