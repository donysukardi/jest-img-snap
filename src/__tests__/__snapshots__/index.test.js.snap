// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index works 1`] = `
Object {
  "checksum": "493044275b17830df3b803d2500dd7ff",
  "filename": "index-works-1-snap.png",
  "type": "image",
}
`;

exports[`index works 2`] = `
Object {
  "checksum": "493044275b17830df3b803d2500dd7ff",
  "filename": "index-works-2-snap.png",
  "type": "image",
}
`;
